data/team.yml

```diff
@@ -121,3 +121,88 @@
       name: LinkedIn
     - link: "https://github.com/realgam3"
       name: Github
+
+- title: Limor Kessem
+  type: cfp
+  subtitle: 
+  key: limor_kessem
+  socials:
+    - link: ""
+      name: Twitter
+    - link: ""
+      name: Site
+    - link: ""
+      name: LinkedIn
```

Comment:
We should have at least everyone's LinkedIn
Can add other socials later on -  but remove empty ones, looks bad just icons

Reply:
it's just a place holder, once merge i'll work with each of them to determine which socials they want in there and will remove the empty ones